op_crates/crypto/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+[package]
+name = "deno_crypto"
+version = "0.1.0"
+edition = "2018"
+description = "Collection of WebCrypto APIs"
```

Comment:
@littledivy please modify `.github/ci.yml` to publish this create similar to `deno_web`

Reply:
done